### PR TITLE
[bitnami/prometheus] Add VIB tests

### PR DESCRIPTION
**Pull request by andresbono**
### Description of the change

The main objective of this PR is to publish our Bitnami Prometheus container using VMware Image Builder. In order to do that, several changes are included:

- Increasing the existing test coverage of the asset by adding Goss tests.
- Update verify and publish VIB pipeline's definitions.

### Benefits

- Ensuring higher quality of the container catalog.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could introduce additional flakiness to the CI/CD.

### Applicable issues

NA

### CODE REVIEW

File: .vib/prometheus/goss/prometheus.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+command:
```
Comment (FraPazGal) [inline .vib/prometheus/goss/prometheus.yaml:1]: Looking at the Dockerfile's `ENTRYPOINT & CMD`, I think we can easily check the prometheus server runs correctly by just runnning:

```
timeout --preserve-status 5 prometheus --config.file=/opt/bitnami/prometheus/conf/prometheus.yml
```

Comment (andresbono) [reply .vib/prometheus/goss/prometheus.yaml:1]: Great suggestion! I will try to apply that to the other PRs when possible.

**Comment by andresbono** [comment]
Tests passing: https://github.com/bitnami/containers/actions/runs/5005181796/jobs/8968775023

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**